src/Properties/BadStatesProperty.jl

```diff
@@ -70,3 +71,8 @@ Let ``Y`` be the bad states represented by 𝑃.
 @inline function check(𝑃::BadStatesProperty, X::LazySet; witness::Bool=false)
     return isdisjoint(X, 𝑃.bad, witness)
 end
+
+@inline function project(𝑃::BadStatesProperty, vars::Vector{Int})
+    proj = Approximations.project(𝑃.bad, vars)
```

Comment:
```suggestion
    proj = project(𝑃.bad, vars)
```
This should not be necessary.

Reply:
it doesn't work without specifying the module. Somehow it is looking for `Approximations` module in `solve.jl` and for  `Properties` module inside `BadStatesProperty` and `SafeStatesProperty`.

Reply:
This is because you did not put the import in the line I pointed to. It must come inside the module. I will fix it.